nipype/utils/filemanip.py

```diff
@@ -229,38 +222,19 @@ def check_forhash(filename):
         return False, None
 
 
-def hash_infile(afile, chunk_len=8192, crypto=hashlib.md5,
-                raise_notfound=False):
-    """
-    Computes hash of a file using 'crypto' module
-
-    >>> hash_infile('smri_ants_registration_settings.json')
-    '49b956387ed8d95a4eb44576fc5103b6'
-
-    >>> hash_infile('surf01.vtk')
-    'fdf1cf359b4e346034372cdeb58f9a88'
-
-    >>> hash_infile('spminfo')
-    '0dc55e3888c98a182dab179b976dfffc'
-
-    >>> hash_infile('fsl_motion_outliers_fd.txt')
-    'defd1812c22405b1ee4431aac5bbdd73'
-
-
-    """
-    if not op.isfile(afile):
-        if raise_notfound:
-            raise RuntimeError('File "%s" not found.' % afile)
-        return None
-
-    crypto_obj = crypto()
-    with open(afile, 'rb') as fp:
-        while True:
-            data = fp.read(chunk_len)
-            if not data:
-                break
-            crypto_obj.update(data)
-    return crypto_obj.hexdigest()
+def hash_infile(afile, chunk_len=8192, crypto=hashlib.md5):
+    """ Computes hash of a file using 'crypto' module"""
+    hex = None
+    if op.isfile(afile):
+        crypto_obj = crypto()
+        with open(afile, 'rb') as fp:
+            while True:
+                data = fp.read(chunk_len)
+                if not data:
+                    break
+                crypto_obj.update(data)
+        hex = crypto_obj.hexdigest()
+    return hex
```

Comment:
If we're dropping the `raise_notfound` check, I'd prefer to leave it:

```Python
if not op.isfile(afile):
    return None

crypto_obj = crypto()
...
```

This checks against bad input, and then runs the function as normal, rather than wrapping the entire behavior in an if block. I don't see a compelling reason to move from this approach.

If we need to move to a big `if` statement, you should not use the keyword `hex` as a variable name.